test/gemm-microkernel-tester.h

```diff
@@ -354,6 +354,12 @@ class GemmMicrokernelTester {
                  xnn_pack_weights_and_biases_fn pack,
                  xnn_packed_stride_weights_and_biases_fn packed_stride);
 
+void Test_PF16(xnn_packed_f16_lhs_igemm_ukernel_fn packed_igemm,
```

Comment:
For this function to be called, you need to add your new kernel to the corresponding `.yaml` files and likely also add a few lines to the test generator script (see e.g. `XNNPACK/test/qp8-f32-qc8w-gemm-minmax.cc`).